src/factories/routeFactory.ts

```diff
@@ -7,8 +7,22 @@ export interface IRouteResult{
     route?: IRoute;
 }
 
+interface IExternalRoute {
+    lintunnus: string;
+    reikuka: string;
+    reinimi: string;
+    reinimilyh: string;
+    reinimilyhr: string;
+    reinimir: string;
+    reitinsuuntasByReitunnus: {
+        edges: Object[],
+    };
+    reitunnus: string;
+    reiviimpvm: string;
+}
+
 class RouteFactory {
-    public static createRoute = (reitti: any, line?: ILine): IRouteResult => {
+    public static createRoute = (reitti: IExternalRoute, line?: ILine): IRouteResult => {
         const routePathResults:IRoutePathResult[]
             = reitti.reitinsuuntasByReitunnus.edges
                 .map((routePath: any, index:number) => {
```

Comment:
Change this to use a type too.

Reply:
Outdated, I added type there:

.map((routePath: IExternalRoutePath) => {

resolving